app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -39,7 +39,7 @@ import org.simple.clinic.util.UuidRoomTypeConverter
       Appointment::class,
       Communication::class,
       MedicalHistory::class],
-    version = 14,
+    version = 15,
```

Comment:
Can you please ensure the schema diff is checked in along with the version change?

Reply:
done